app/Http/Controllers/API/ScenariosController.php

```diff
@@ -132,23 +132,32 @@ private function createDefaultConversations(Scenario $scenario): Scenario
         $welcomeOutgoingIntentId = "intent.app.welcomeResponseFor$scenarioNameAsId";
         $noMatchOutgoingIntentId = "intent.app.noMatchResponse$scenarioNameAsId";
 
+
+        $incomingIntentId = "intent.$scenarioNameAsId.autogenerated";
```

Comment:
Nothing major at all - just wondering whether it might keep things simply to use the `app` namespace here instead.

```suggestion
        $incomingIntentId = "intent.app.autogenerated";
```